dice_ml/explainer_interfaces/feasible_model_approx.py

```diff
@@ -81,8 +81,9 @@ def train(self, constraint_type, constraint_variables, constraint_direction, con
             train_loss = 0.0
             train_size = 0
 
-            train_dataset = torch.tensor(self.vae_train_feat).float()
-            train_dataset = torch.utils.data.DataLoader(train_dataset, batch_size=self.batch_size, shuffle=True)
+            train_dataset = torch.utils.data.DataLoader(
+                torch.tensor(self.vae_train_feat).float(), # type: ignore
```

Comment:
Set `type: ignore` because the error depends on the implementation of `torch.utils.data.DataLoader`.